status-go-version.json

```diff
@@ -1,9 +1,9 @@
 {
     "_comment": "THIS SHOULD NOT BE EDITED BY HAND.",
     "_comment": "Instead use: scripts/update-status-go.sh <rev>",
-    "owner": "status-im",
+    "owner": "qfrank",
     "repo": "status-go",
-    "version": "v0.110.3",
-    "commit-sha1": "506921509e477445a95e1961a2557ef584d7caec",
-    "src-sha256": "1xmngbfl67n031rxhhbs7m14gzmfhgr1d34szclf1a6qxb9kw946"
+    "version": "improvement/13888",
+    "commit-sha1": "a1ffe1047c0be720649da8d256c7bbc0aa7b8b48",
+    "src-sha256": "1idam71wvgmip48pkg0j47gdf2n58byhw7nzp4f166y883x451ci"
```

Comment:
We should depend on `status-im/status-go`
Seems this is blocked by https://github.com/status-im/status-go/pull/2840

Reply:
yeah, but it's a temp solution, will update after QA test pass